### PR TITLE
revise the english readme documentation

### PR DESCRIPTION
**Pull request by JamesBie**
-remove empty words for more concise wording
    -fix grammar
    -change hesitance to hesitation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,9 @@
 mobile heterogeneous computing platforms. The design focuses on the following
 targets:
 * Performance
-  * The runtime is highly optimized with NEON, OpenCL and Hexagon, and
+  * Tuntime is optimized with NEON, OpenCL and Hexagon, and
```
Comment (l2dy) [inline README.md]: Tuntime?

Comment (l2dy) [reply README.md]: @JamesBie You should amend your commit and force push to this `README` branch.

Comment (JamesBie) [reply README.md]: @l2dy  i've amended my commit, pushed and submitted another pull request

Comment (llhe) [reply README.md]: @JamesBie Thanks for the patch.
Is this deprecated by https://github.com/XiaoMi/mace/pull/34?
Actually, you can use `git rebase HEAD~2` to squash top 2 commits and force to push to `JamesBie:README` branch.

Comment (JamesBie) [reply README.md]: @llhe  Yes its depreciated by #34 so you could close this. Thanks for the tip! I'll be sure to do that next time and sorry about that!
